sys-apps/hd-idle/hd-idle-1.05.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+DESCRIPTION="Utility for spinning down hard disks after a period of idle time"
+HOMEPAGE="http://hd-idle.sourceforge.net/"
+SRC_URI="mirror://sourceforge/${PN}/${P}.tgz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+S=${WORKDIR}/${PN}
+
```

Comment:
Remove `dodoc` in the `src_install` phase and define the `DOCS` array with the files `debian/changelog` and `README` in it.